src/components/app.js

```diff
@@ -21,12 +21,19 @@ export default React.createClass({
     this.setState({user});
   },
   render() {
-    //return <Circle user={this.state.user} />;
+    return <Circle user={this.state.user} onPress={ () => {
+        if (!this.currentUser.ref) return <View />;
+        this.props.navigator.push({
+          title: "Teams",
+          component: Teams,
+          passProps: {
+            userRef: this.currentUser.ref,
+            onPress: this.goToTeam,
+          },
+        });
```

Comment:
Also, given that this is handled inside the component in `Teams`, I reckon this should be moved into the `Circle` component, but I can do that if you like